drafts/inlinemacros.md

```diff
@@ -144,6 +148,7 @@ positions:
 
         inline val x: inline T = E
 
+
```

Comment:
If I got it right, we never infer if type of definition is `inline`. As `inline` has different guarantees with respect to binary compatibility and runtime behavior and subtyping.